code/datums/ruins.dm

```diff
@@ -86,3 +86,11 @@
 /// Returns `TRUE` if this ruin can spawn in current sector, otherwise `FALSE`.
 /datum/map_template/ruin/proc/spawns_in_current_sector()
 	return (SSatlas.current_sector.name in sectors)
+
+/datum/map_template/ruin/proc/port_of_call_check()
```

Comment:
this shouldn't be a ruin proc, this should either be on SSatlas, on the mapdatum, or on the sector - probably on SSatlas